packages/web-client/app/components/card-pay/layer-two-connect-card/index.hbs

```diff
@@ -2,16 +2,12 @@
   @suppressHeader={{@suppressHeader}}
   @header="Wallet Connection - xDai Chain"
   @isComplete={{@isComplete}}
-  class={{if @includeExplanations 'layer-two-connect-card--with-explanations'}}
   data-test-layer-2-wallet-card
   ...attributes
 >
   {{#if (eq this.cardState 'memorialized')}}
-    <section class="layer-two-connect-card__body">
-      <header aria-label="Layer 2 Connected Header" class="layer-two-connect-card__body-header">
-        Cardstack
-        <img class="layer-two-connect-card__body-header-logo" src={{this.cardstackLogo}} alt="" role="presentation">
-      </header>
+    <ActionCardContainer::Section @title="Cardstack" class="layer-two-connect-card__body">
+      <img class="layer-two-connect-card__logo" src={{this.cardstackLogo}} alt="Cardstack" width="60" height="60">
```

Comment:
Similar to the layer one connect card, I think this image shouldn't have alt/be detected by screenreaders.